client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-conflicts.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { translate } from 'i18n-calypso';
```

Comment:
You shouldn't import `translate` directly into your components - instead, you should use `localize` to provide the i18n functions via props. See the [docs](https://github.com/Automattic/i18n-calypso#localize) for more info.